### ISSUE TITLE
literature reference

### ISSUE THREAD

**Issue by Marchphy** [Harry24k/bayesian-neural-network-pytorch#5]
Hi,

Thanks for your work! This library is easy to use and works well on my dataset(big and complex).

Since I am new to BNN, could you please share with me the paper(s) corresponding to your implementation?

**Comment by Harry24k** [comment]
I read following papers 
- Dropout Inference in Bayesian Neural Networks with Alpha-divergences(https://arxiv.org/abs/1703.02914)
- Adv-BNN: Improved Adversarial Defense through Robust Bayesian Neural Network(https://arxiv.org/abs/1810.01279)
- Understanding Measures of Uncertainty for Adversarial Example Detection(https://arxiv.org/abs/1803.08533)
- Dropout as a Bayesian Approximation:  Representing Model Uncertainty in Deep Learning(https://arxiv.org/abs/1506.02142)

I hope these information helps you :)
